packages/default-theme/helpers/index.js

```diff
@@ -16,10 +16,23 @@ export function formatPrice(price, options) {
   ).format()
 }
 
+const getSortingLabel = (sorting) => {
+  if (!sorting || !sorting.order || !sorting.field) {
+    return ''
+  }
+
+  const ascLabel = 'ascending'
+  const descLabel = 'descending'
+
+  const label = sorting.order === 'desc' ? descLabel : ascLabel
+  return `${sorting.field} ${label}`
+}
+
 const formatDate = (date, format = `DD-MM-YYYY H:m:s`) =>
   dayjs(date).format(format)
 
 export default {
```

Comment:
let's remove this default export and check if it's somewhere used that way. With default export, there is no tree-shake